styles.css

```diff
@@ -46,12 +51,18 @@ body {
 
 /*  ---- start exercise 3 ----  */
 .card-3 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
`justify-content: space-between` isn't doing anything because you are then aligning flex items "manually" (using the `margin-top`)